pkg/cmd/admin/router/router.go

```diff
@@ -325,6 +325,9 @@ func generateSecretsConfig(cfg *RouterConfig, kClient *kclient.Client,
 		mounts = append(mounts, mount)
 	}
 
+	// default cert (which is a pem) is provided.
```

Comment:
"When the user provides a default cert it is in pem format.
If the default cert is not provided by the user, we generate one using a service annotation, but that will have a cert and a key we need to combine into the pem that haproxy expects"